kuka_iiqka_eac_driver/config/ros2_controller_config.yaml

```diff
@@ -14,3 +14,5 @@ controller_manager:
       type: kuka_controllers/ControlModeHandler
     event_broadcaster:
       type: kuka_controllers/EventBroadcaster
+    twist_controller:
```

Comment:
I think there should also be a separate yaml for the KUKA_MR specific controllers